pkg/kube/controllers/quarksjob/errand_reconciler.go

```diff
@@ -84,9 +86,33 @@ func (r *ErrandReconciler) Reconcile(request reconcile.Request) (reconcile.Resul
 		return reconcile.Result{}, err
 	}
 
-	if meltdown.NewWindow(r.config.MeltdownDuration, qJob.Status.LastReconcile).Contains(time.Now()) {
-		ctxlog.WithEvent(qJob, "Meltdown").Debugf(ctx, "Resource '%s' is in meltdown, requeue reconcile after %s", request.NamespacedName, r.config.MeltdownRequeueAfter)
-		return reconcile.Result{RequeueAfter: r.config.MeltdownRequeueAfter}, nil
+	// negativeTime is used here to ensure that the time
+	// is not same as now() otherwise meltdown implementation
+	// will fail.
+	negativeTime := metav1.Date(0001, 1, 1, -1, 0, 0, 0, time.UTC)
```

Comment:
this could be a constant in the meltdown package

Reply:
How do you think ReconcileSkipDuration is ?

Reply:
negativeTime is more like a marker, not a duration :)

Reply:
Done. Spoke to you on call.